src/_posts/databases/postgresql/2000-01-01-upgrading.md

```diff
@@ -0,0 +1,233 @@
+---
+title: Upgrading Your Scalingo for PostgreSQL® Addon
+nav: Upgrading
+modified_at: 2024-11-27 12:00:00
+tags: databases postgresql addon
+index: 6
+---
+
+In Scalingo terminology, ***upgrading*** a Scalingo for PostgreSQL® addon
+designates the operation consisting in changing the database version for a
+newer one.
+
+We disinguish two main cases:
+- a ***minor-upgrade*** designates an incremental release with a change in the
+  tenths digit of the version number (e.g. `13.3.4` to `13.5.0`). It usually
+  provides bugs or security fixes as well as minor additional features.
+- a ***major-upgrade*** designates an incremental release with a change in the
+  main version number (e.g. `13.x.y` to `14.0.0`). Major-upgrades often provide
+  significant changes and new features that **can** break backward
+  compatibility.
+
+While we usually advise to stick to the latest minor-upgrade available to
+benefit from bug and security fixes, we also **strongly** advise to take
+extra-care when major-upgrading your Scalingo for PostgreSQL® addon ([more
+about this below](#best-practices-when-managing-major-upgrades)).
+
+When the database vendor releases a new version of your database engine, we
+take some time to study it and test it thoroughly before making it available.
+Upgrading to this new version is still your choice. We never do it
+automatically.
```

Comment:
> We never do it automatically

IMO that's wrong. For example, in the context of PostgreSQL 12 end of life, we automatically upgrade databases if the owner does not do it on their own.

Reply:
Removed in #2885